test/unit/recipe/session/utils.unit.test.ts

```diff
@@ -0,0 +1,66 @@
+import { getFailureRedirectionInfo } from "../../../../lib/ts/recipe/session/utils";
+import * as utils from "supertokens-web-js/utils";
+
+beforeEach(() => {
+    jest.spyOn(utils, "getGlobalClaimValidators").mockReturnValue([
+        {
+            id: "id-3",
+            onFailureRedirection: undefined,
+        },
+        {
+            id: "id-2",
+            onFailureRedirection: () => undefined,
+        },
+        {
+            id: "id-1",
+            onFailureRedirection: () => "/id-1",
+        },
+    ] as any[]);
+});
+
+describe(`${getFailureRedirectionInfo.name} functionality`, () => {
```

Comment:
Add a test-case that shows that the order of redirection depends on `getGlobalClaimValidators`: two claims should return a string, the one that comes first in `getGlobalClaimValidators` should be used.